pylatex/base_classes/containers.py

```diff
@@ -178,3 +178,61 @@ def dumps(self):
         string += Command('end', self.latex_name).dumps()
 
         return string
+
+
+class ContainerCommand(Container):
+    r"""A base class for a container command (A command which contains data).
+
+    Container command example:
```

Comment:
If you add the following in between these lines Sphinx error should be gone (add a newline before and after):
```rst

.. code-block:: latex

```

Is a special package needed for the textcolor command.